apps/docs/pages/core/modules/custom.mdx

```diff
@@ -0,0 +1,73 @@
+import { Callout, Steps } from "nextra/components";
+
+# Using the Custom Module in Wallet API Client
+
+The Custom Module allows you to interact with any custom handler supported by the server.
+
+## Custom Module Overview
+
+<Callout type="info" emoji="ℹ️">
+  We are still working on improving the type safety of custom modules
+</Callout>
+
+<Steps>
+### Step 1
+
+Create the WalletAPIClient with the `getCustomModule` param.
+
+```javascript
+import {
+  CustomModule,
+  WalletAPIClient,
+  defaultLogger,
+} from "@ledgerhq/wallet-api-client";
+
+type CustomLogHandlersType = {
+  "custom.log": (message: string) => Promise<{ res: "hello" }>;
+};
+
+class CustomLog extends CustomModule {
```

Comment:
This should definitely be explained.

"To create a Custom Handler, you need to import and extend the CustomModule interface, it should return the request that..."

Reply:
I'll explain more